account/account_test.go

```diff
@@ -955,6 +955,50 @@ func TestAddDeclareTxn(t *testing.T) {
 	}
 }
 
+func TestDeployAccountDevnet(t *testing.T) {
```

Comment:
This doesn't run for me (eg see wrapping tx type in rpc.BroadcastDeployAccountTxn comment above)

Reply:
I need to merge the new RPCv0.5 with my branch, this applies to other parts of my code